pkg/debug/spec.sh

```diff
@@ -72,6 +72,8 @@ else
 fi
 
 DISK=$(lsblk -b  | grep disk | awk '{ total += $4; } END { print int(total/(1024*1024*1024)); }')
+WDT=$([ -e /dev/watchdog ] && echo true || echo false)
+HSM=$([ -e /dev/tpm0 ] && echo 1 || echo 0)
```

Comment:
/dev/tpm0 is TPM 1.2.
Need to look for /dev/tpmrm0 for TPM 2.0

Reply:
Fixed